### DISCUSSION TITLE
Strikethrough text line rendering too high

### DISCUSSION THREAD

**Discussion by simonmandlik** [wezterm/wezterm#4196] [Q&A] [answered]
I'm using `FiraCode Nerd Font` with

```lua
config.line_height = 1.2
```

This is how strikethrough text looks in wezterm

<img width="222" alt="image" src="https://github.com/wez/wezterm/assets/16707112/42800a6e-6795-4fd4-8b00-349efe477446">

Seems like the position of the line is computed not from the text height, but from the cell height. When `line_height` is changed, the line is above the actual vertical center.

For comparison, this is how the same text looks in iterm2

<img width="222" alt="image" src="https://github.com/wez/wezterm/assets/16707112/9279963c-c971-44b8-b1bf-ebda587beb5f">

Should I report this as an issue or is this intended behavior?

Thanks!

**Comment by wez** [comment, marked as answer]
Fixed in 360ad2a3a975f86697c7f6f5327d216b5799f5b2
